cpp/src/cider/type/data/sqltypes.h

```diff
@@ -70,6 +70,8 @@ enum SQLTypes {
   kCOLUMN = 27,
   kCOLUMN_LIST = 28,
   kSTRUCT = 29,
+  kINT128 = 30,
+  kINT256 = 31,
   kSQLTYPE_LAST
```

Comment:
internal type? I think it's not sql type.

Reply:
Well, it's about the naming, right? Maybe we can call large int or huge int etc.

Reply:
`kINT128` and `kINT256` will be used as key of aggregation and they're new column types.

Reply:
I think it belongs to type implementation. better use get_logical_size() to identify internal type？

Reply:
need to update sqltypeInfo in sqltypes.h?

Reply:
I guess not in this PR, since we can add this while supporting Int128&Int256 for the whole workflow.